internal/balancer/gracefulswitch/gracefulswitch.go

```diff
@@ -293,6 +294,7 @@ func (gsb *Balancer) Close() {
 // State updates from the wrapped balancer can result in invocation of the
 // graceful switch logic.
 type balancerWrapper struct {
+	balancer.ClientConn
```

Comment:
If we are embedding now, we can remove the `Target` implementation.

Reply:
Removed.